cloudify/amqp_client.py

```diff
@@ -476,8 +476,15 @@ def _log_message(self, message):
         level = message.get('level', 'info')
         log_func = getattr(self.logger, level, self.logger.info)
         exec_id = message.get('context', {}).get('execution_id')
+        execution_creator_username = message.get('context', {}).get(
```

Comment:
This might also benefit from the logging changes we're discussing in the other PR.